spec/unit/target/build_settings/aggregate_target_settings_spec.rb

```diff
@@ -566,7 +566,7 @@ def pod_target(spec, target_definition)
             pod_target.stubs(:build_settings => PodTargetSettings.new(pod_target))
             aggregate_target = fixture_aggregate_target([pod_target])
             @generator = AggregateTargetSettings.new(aggregate_target, 'Release')
-            @generator.other_ldflags.should == %w(-ObjC -l"StaticLibrary" -l"VendoredDyld" -l"xml2" -framework "PodTarget" -framework "StaticFramework" -framework "VendoredFramework" -framework "XCTest")
```

Comment:
why is this no longer linking libxml, which is a system library?

Reply:
it matches 1.5.0 build settings. With 1.6.0.beta we were adding system frameworks to aggregate targets that were meant to be only for pod targets. It works but I believe it was redundant.

Reply:
well because it matches 1.5.0 did. Dynamic frameworks are probably unnecessary but static libs *should* be there since CocoaPods since the beginning has been designed to link dependencies to the target app not to the framework that requires them only (which is a no-op for cases that don't use `use_frameworks!`)